.github/workflows/update.yml

```diff
@@ -14,12 +14,4 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@main
-      - uses: denoland/setup-deno@main
-        with:
-          deno-version: v1.x
-      - name: Update dependencies
-        run: deno run -A https://deno.land/x/update/mod.ts -b
```

Comment:
simply changing this to `deno run -A --import-map=https://deno.land/x/update/deno.json https://deno.land/x/update/mod.ts -b` would have worked as well

Reply:
We don't take the easy path here